### DISCUSSION TITLE
How to access the entities in NER.v2 in spacy-llma from logging or directly as they come from GPT-3 without using .ents ?

### DISCUSSION THREAD

**Discussion by innocent-charles** [explosion/spaCy#12845] [Help: Coding & Implementations] [answered]
i want to access and take the entities from logging or directly from gpt-3 without using .ents . 

Example: consider the labels of PERSON and LOCATION . The text such as "The woman Paris was walking around in Paris"

1. from LLM logging 
DEBUG:spacy_llm:LLM response for doc: The woman Paris was walking around in Paris
LOCATION: Paris
PERSON: Paris

2. but from .ents 
LOCATION: Paris
LOCATION: Paris
 

So, that means there is problem when taking response from LLM to Doc ... 

now, i want a way to access the entities directly from LLMs or from logging maybe , i don't want to use .ents

**Comment by rmitsch** [comment]
Hi @innocent-charles, this looks like a bug in the `spacy-llm` NER task. We'll look into that as soon as possible.
As for the LLM response, the best way to access this is via your logs (you can define your logger to write to file).

**Reply by rmitsch** [reply, marked as answer]
I can confirm this is a bug. I'll update this thread once we implemented a fix and released a patched version of `spacy-llm`.

**Comment by svlandeg** [comment]
(Duplicate of https://github.com/explosion/spaCy/discussions/12812)
